docs/source/relayer.rst

```diff
@@ -243,31 +188,115 @@ Example:
     cool-domain.test owner set to: 0xf1Af1CCEEC4464212Fc7b790c205ca3b8E74ba67 (registrar)
 
 
-Register ENS name
-^^^^^^^^^^^^^^^^^
 
-To register ENS name, in universal-login-relayer directory type in the console:
+Programmatically
+^^^^^^^^^^^^^^^^
+
+To register own ENS domain programmatically, you should use DomainRegistrar. 
 
-  ::
+**new DomainRegistrar(config)**
+  creates DomainRegistrar.
 
-    yarn register:name name my-domain.test
+  Parameters: 
+    - **config** : object - specific config parameters, includes: 
 
-Parameters:
-  - **name** - name to register
-  - **my-domain.test** - existing domain
+      - **jsonRpcUrl** : string - JSON-RPC URL of an Ethereum node
+      - **privateKey** : string - private key to execute registrations
+      - **ensAddress** : string - address of ENS
+      - **publicResolverAddress** : string - address of public resolver
+  Returns:
+    DomainRegistrar instance
 
-Example:
-  ::
+  Example:
+    ::
+
+      const ensRegistrationConfig = {
+        jsonRpcUrl: 'https://ropsten.infura.io',
+        privateKey: 'YOUR_PRIVATE_KEY',
+        chainSpec: {
+          ensAddress: '0x112234455c3a32fd11230c42e7bccd4a84e02010',
+          publicResolverAddress: '0x4C641FB9BAd9b60EF180c31F56051cE826d21A9A',
+          chainId: 0
+        }
+      }
+      const registrar = new DomainRegistrar(ensRegistrationConfig);
+
+**registrar.registerAndSave(domain, tld)** 
+  registers new domain and saves to new file all informations about newly registered domain (registrar address or resolver address)
+
+  Parameters:
+    - **domain** : string - domain to register
+    - **tld** : string - top level domain, for example: ``eth`` or on testnets: ``test``
+
+  Example:
+    ::
+
+      registrar.registerAndSave('new-domain', 'test');
 
-    yarn register:name justyna cool-domain.test
 
-  Result 
+.. _custom-relayer:
+
+Custom relayer
+--------------
+
+Create custom relayer to grant tokens or ether.
```

Comment:
You can subclass relayer to create custom behavior, e.g. a relayer that grants ether or tokens to a newly created wallet contract.